test/onnx/test_pytorch_onnx_onnxruntime_cuda.py

```diff
@@ -85,6 +85,20 @@ def forward(self, x):
         model = amp.initialize(LinearModel(), opt_level="O2")
         self.run_test(model, input)
 
+    # ONNX supported bfloat16 for opsets >= 13
+    @skipIfUnsupportedMinOpsetVersion(13)
+    @skipIfNoBFloat16Cuda
+    @disableScriptTest()
+    def test_linear_bfp16(self):
+        class MyModule(torch.nn.Module):
+            def forward(self, x):
```

Comment:
please add a comment explaining what in this model can't be tested on CPU.
If everything here can be tested on CPU, then just remove the test.

Reply:
Added the comment.